chat-commands.js

```diff
@@ -1464,7 +1464,7 @@ exports.commands = {
 		if (!room.users[targetUser.userid]) {
 			return this.errorReply("User " + this.targetUsername + " is not in the room " + room.id + ".");
 		}
-		if (targetUser.joinRoom(targetRoom.id) === false) return this.errorReply("User " + targetUser.name + " could not be joined to room " + targetRoom.title + ". They could be banned from the room.");
+		if (targetUser.joinRoom(targetRoom.id) === null) return this.errorReply("User " + targetUser.name + " could not be joined to room " + targetRoom.title + ". They could be banned from the room.");
```

Comment:
This would allow people to be redirected to staff rooms regardless of if the user's staff for not. I'd just check if the return value's falsey